server/models/meetup.js

```diff
@@ -0,0 +1,72 @@
+const meetup = [{
+  id: 3,
+  createdOn: new Date(),
+  location: 'aduwawa',
+  images: 'this is the imgae url',
+  topic: 'This is the best meetup ever ',
+  happeningOn: new Date(2019, 2, 1),
+  tags: ['business', 'entertianment'],
+  description: 'this is a very short description of this very second meetup',
+  noOfQuestions: 2,
+}];
+
+// create a class to handle the meetup array
```

Comment:
Change this to a [JSDOCS](http://usejsdoc.org/) comment.

Reply:
Hello chima
I have checked comment to jsdocs now
Thanks